packages/grid/_modules_/grid/models/gridExport.ts

```diff
@@ -1,7 +1,21 @@
+export interface GridExportCsvOptions {
+  utf8WithBom?: boolean;
+  disabled?: boolean;
```

Comment:
```suggestion
```

Reply:
Change applied, but we should find another solution to set which format enable or disable (csv | pdf | excel).